src/LibraryManager.Vsix/UI/Controls/MutualPropertyChange.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.ComponentModel;
+using Microsoft.Web.LibraryManager.Vsix.UI.Models;
+
+namespace Microsoft.Web.LibraryManager.Vsix.UI.Controls
+{
+    internal class MutualPropertyChange : BindableBase
```

Comment:
>MutualPropertyChange [](start = 19, length = 20)

This needs a comment explaining what it is for. It's not obvious from the code.

Also, it should have a more descriptive name. Maybe "BindLibraryNameToTargetLocation". It doesn't hav eto be that, but choose a name so when I look at the class I know what it's doing.

Reply:
Will do

---
In reply to: [200770461](https://github.com/aspnet/LibraryManager/pull/285#discussion_r200770461) [](ancestors = 200770461)